### DISCUSSION TITLE
Backlog

### DISCUSSION THREAD

**Discussion by davidfrantz** [davidfrantz/force#193] [General]
Dear FORCE users,

thank you all for your long-lasting interest in this software project!

There is quite some backlog on my end regarding open issues and discussions. Apologies for that!

Open-source software is a passion of mine. And I am really happy that this project took off and that a small community has developed! Also many thanks to all regular contributors (esp. @ernstste)!

Writing, maintaining, and administering an open-source project is a lot of additional work, though. This work usually comes on top of the daily duties of my 100+% job, as well as family. Therefore, there will always be periodic times when I cannot manage to put time into this project. My apologies for that, but rest assured, the FORCE is still living, and I am fairly confident that I will find some time soon to at least answer most of the issues.

Next week will be the Living Planet Symposium. In case I will end up sitting in some boring sessions, I might find some time there.

Cheers,
David

**Comment by mhelleis** [comment]
Dear David, this is completely understandable. Thank you for the amazing work you (and others) did on FORCE so far!
